src/App.js

```diff
@@ -21,7 +21,7 @@ class App extends Component {
             <a href="/">
               <img
                 className="cfd-logo"
-                src="images/cfd-circle-white.png"
+                src="../images/cfd-circle-white.png"
```

Comment:
perfectly fine as long as we keep directory structure consistent.

Reply:
I just took a look at out current folder structure and it seems like you would need to change that line to: `"../public/images/cfd-circle-white.png"`

So curious how the current change is working... Let me know if this makes sense.

Reply:
@KeynesYouDigIt this doesn't work for routes that are more than 2 slashes deep. See [/a/a/a](https://deploy-preview-119--cfd-members.netlify.com/a/a/a) as an example. The way that `create-react-app` is set up builds all the static files in `public` into the root of what gets served. This means that instead of using `../` you can just use `/images/cfd-circle-white.png` and it should work for all the cases.

Reply:
Actually, the other option for this, that `create-react-app` recommends it to utilize webpack and `import` the image directly into the javascript file. [See this for info](https://github.com/facebook/create-react-app/blob/master/packages/react-scripts/template/README.md#adding-images-fonts-and-files)

Reply:
[More info on when to use the `public folder`](https://github.com/facebook/create-react-app/blob/master/packages/react-scripts/template/README.md#when-to-use-the-public-folder)

Reply:
Well derp. youre right. updates pending.........